ui/shared/auth/AuthService.js

```diff
@@ -90,17 +103,8 @@ export default class AuthService {
   }
 
   logout() {
+    this.usingThis = null; // Just for using this
```

Comment:
hahaha 🤣 
Could you please use `//eslint-disable-next-line rule-name` instead?

Reply:
Whoops I missed that one!